### PR TITLE
User,Product modules fixed

### PR DESCRIPTION
**Pull request by svelamuri26**

### CODE REVIEW

File: src/main/java/com/example/NiRi/modules/LoginRequest.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+     public String getPassword() {
+            return password;
+        }
+
+     public void setPassword(String password) {
+            this.password = password;
+        }
+
+}*/
```
Comment (MenilV) [inline src/main/java/com/example/NiRi/modules/LoginRequest.java]: do we still need this?

File: src/main/java/com/example/NiRi/service/OrderService.java [hunk 2]
```diff
@@ -78,5 +81,20 @@ private void markCartItemsAsPurchased(List<CartItem> cartItems, Order order) {
...
+        User user = userRepository.findByEmail(userEmail)
+                .orElseThrow(() -> new RuntimeException("User not found with email: " + userEmail));
+
+        return orderRepository.findByUser(user);
+    }
+
+    public Order createOrder(OrderCreationRequest orderRequest) {
+
+        Order newOrder = new Order();
+        newOrder.setUser();
```
Comment (MenilV) [inline src/main/java/com/example/NiRi/service/OrderService.java]: is this the correct way?

**Review by MenilV — CHANGES_REQUESTED** [review]
a few things to check first.

File: src/main/java/com/example/NiRi/modules/LoginRequest.java (hunk 1 above)
Comment (svelamuri26) [reply src/main/java/com/example/NiRi/modules/LoginRequest.java]: I left it as it is to add authentication sometime later after all the modules are completed

File: src/main/java/com/example/NiRi/service/OrderService.java (hunk 2 above)
Comment (svelamuri26) [reply src/main/java/com/example/NiRi/service/OrderService.java]: I´ve changed it and now the issue is fixed

**Review by faruka-startsteps — APPROVED** [review]

**Pull request merged**